draft-trammell-taps-interface.md

```diff
@@ -1073,117 +859,117 @@ decrease.
 
 #### Ordered {#send-ordered}
 
-Ordered is a boolean property. If true, this Content should be delivered after
-the last Content passed to the same Connection via the Send action; if false,
-this Content may be delivered out of order.
-
-#### Immediate {#send-immediate}
-
-Immediate is a boolean property. If true, the caller prefers immediacy to
-efficient capacity usage for this Content. For example, this means that
-the Content should not be bundled with other
-Content into the same transmission by the underlying Protocol Stack.
+Ordered is a boolean property. If true, this Message should be delivered after
+the last Message passed to the same Connection via the Send action; if false,
+this Message may be delivered out of order.
 
 #### Idempotent {#send-idempotent}
 
 Idempotent is a boolean property. If true, the application-layer entity in the
-Content is safe to send to the remote endpoint more than once for a single
+Message is safe to send to the remote endpoint more than once for a single
 Send action. It is used to mark data safe for certain 0-RTT establishment
 techniques, where retransmission of the 0-RTT data may cause the remote
-application to receive the Content multiple times.
+application to receive the Message multiple times.
 
 #### Corruption Protection Length {#send-checksum}
 
-This numeric property specifies the length of the section of content, starting
-from byte 0, that the application assumes will be received without corruption
-due to lower layer errors. It is used to specify options for simple integrity
-protection via checksums. By default, the entire Content is protected by
-checksum. A value of 0 means that no checksum is required, and a special value
-(e.g. -1) can be used to indicate the default. Only full coverage is
+This numeric property specifies the length of the section of the Message,
+starting from byte 0, that the application assumes will be received without
+corruption due to lower layer errors. It is used to specify options for simple
+integrity protection via checksums. By default, the entire Message is protected
+by checksum. A value of 0 means that no checksum is required, and a special
+value (e.g. -1) can be used to indicate the default. Only full coverage is
 guaranteed, any other requests are advisory.
 
 #### Immediate Acknowledgement {#send-ackimmed}
 
 This boolean property specifies, if true, that an application wants this
-Content to be acknowledged immediately by the receiver. In case of reliable
+Message to be acknowledged immediately by the receiver. In case of reliable
 transmission, this informs the transport protocol on the sender side faster
-that it can remove the Content from its buffer; therefore this property can be
+that it can remove the Message from its buffer; therefore this property can be
 useful for latency-critical applications that maintain tight control over the
 send buffer (see {{sending}}).
 
-#### Send Bitrate {#send-bitrate}
+#### Instantaneous Capacity Profile
```

Comment:
Interesting idea to make this whole choice per-message. I'm not 100% sure about everything here - e.g. the value choices ("constant rate" in particular). But we should make progress, and I think this is a fine discussion to postpone until after London.

Reply:
yeah i kind of hate "instantaneous capacity profile" as a name ("timeliness" isn't great either, though"); inclined to leave this as is and revisit after we decide on #90.

Reply:
Sure, I agree